.github/workflows/tests.yml

```diff
@@ -133,6 +133,7 @@ jobs:
 
       - name: Start MongoDB
         run: |
+          reg add HKLM\System\CurrentControlSet\Control /v ServicesPipeTimeout /t REG_DWORD /d 300000 /f
```

Comment:
This did not have the desired effect, as the service still timed out: https://github.com/mongodb/mongo-php-driver/actions/runs/5109526170/jobs/9184410368?pr=1429#step:8:1

I've opened [PHPC-2233](https://jira.mongodb.org/browse/PHPC-2233) to consider a different approach.